docs/sources/developers/plugins/create-a-grafana-plugin/develop-a-plugin/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Develop a Grafana plugin
+menuTitle: Develop a plugin
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - development
+  - documentation
+description: An index of how-to topics for Grafana plugin development.
+weight: 100
+---
+
+# Develop a Grafana plugin
+
+This section contains how-to topics for developing Grafana plugins.
+
+- [Build a Grafana plugin with the create-plugin tool](https://grafana.github.io/plugin-tools/docs/creating-a-plugin)
+- [Build a panel plugin]({{< relref "./build-a-panel-plugin.md" >}})
+- [Build a panel plugin with d3.js]({{< relref "./build-a-panel-plugin-with-d3.md" >}})
+- [Build a data source plugin]({{< relref "./build-a-data-source-plugin.md" >}})
+- [Build a data source backend plugin]({{< relref "./build-a-data-source-backend-plugin.md" >}})
+- [Build a logs data source plugin]({{< relref "./build-a-logs-data-source-plugin.md" >}})
+- [Build a streaming data source plugin]({{< relref "./build-a-streaming-data-source-plugin.md" >}})
+- [Sign a plugin]({{< relref "./sign-a-plugin.md" >}})
```

Comment:
To me `Sign a plugin` makes more sense in publishing and is where the content was previously merged - so feels like this may be disorientating for users

Reply:
Again not sure if you don't just want `{{< section menuTitle="true">}}` if its purely an index

Reply:
Regarding the "Sign a plugin" location, I seem to remember that my doc plan originally had it in "Publishing", but you and/or Timur requested that it be moved because it wasn't necessarily part of the publishing flow. I am open to either location.

Reply:
The reason I haven't been using {{< section menuTitle="true">}}  is that these landing pages (temporarily) include links to the Plugin Tools repo that wouldn't be generated by Hugo.

Reply:
>Regarding the "Sign a plugin" location, I seem to remember that my doc plan originally had it in "Publishing", but you and/or Timur requested that it be moved because it wasn't necessarily part of the publishing flow. I am open to either location.

publishing is definitely my preference